src/main/java/org/jabref/gui/importer/actions/OpenDatabaseAction.java

```diff
@@ -178,9 +177,7 @@ private void openTheFile(Path file) {
         }
 
         BackgroundTask<ParserResult> backgroundTask = BackgroundTask.wrap(() -> loadDatabase(file));
```

Comment:
Can you please add a comment that the execution of the task happens inside the LIbraryTabe ?
When I first saw this method here without any executor, I thought it is missing here...